app/views/shared/_ML_dev_big_card.html.erb

```diff
@@ -0,0 +1,45 @@
+
+<div style="z-index: 100;">
+    <div class="path-card-expanded mt-4 pt-5 px-5" data-big-card-id="ml_developer">
+      <div>
+        <!-- Cup -->
+        <span class="iconify" data-icon="si-glyph:champion-cup" data-inline="false" style="font-size: 150px; color: #997029;"></span>
+         <h5 class="mt-4">"ML Developer"</h5>
+        <!-- Main info -->
+        <div class="d-flex text-left" style="display: flex;">
+            <i class="far fa-calendar-check step-duration-symbol"></i>
+            <p>duration</p>
+        </div>
+        <div class="d-flex text-left" style="display: flex; align-items: center;">
+          <i class="fas fa-circle"></i><i class="far fa-circle"></i><i class="far fa-circle mr-3"></i>
+          <p> Intermediate</p>
+        </div>
+        <!-- Skills -->
+        <div class="d-flex">
+          <div class= "d-flex skill-icon" style="flex-direction: column; align-items: center;">
+            <span class="iconify" data-inline="false" data-icon="bx:bxl-python" style="font-size: 24px; color: #003366;"></span>
+            <p style="font-size: 15px;">Python</p>
+          </div>
+        </div>
+        <!-- Description -->
+        <div>
+          <p><strong>WHAT YOU FIND IN THIS PATH</strong></p>
+          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.
```

Comment:
@sabinacuccibar do you mean here we have the issue with seed files? can we use here @path.description to make it dynamic?

Reply:
You can ignore this comment. I realised it is because we don't have seeds for other paths.